modules/ROOT/pages/migration-connectors-email.adoc

```diff
@@ -129,13 +129,13 @@ accessible through DataWeave. For instance, to reference an attachment called
 </flow>
 ----
 
-On the Mule 3 transport, the properties of the outgoing e-mail (subject, 
-addresses, etc) could be configured via outbound properties set in the event 
-before the execution of the outbound smtp endpoint. In that case, the outbound 
-properties would take precedence over the values configured in the endpoint or 
-connector themselves.  
+In the Mule 3 transport, the properties of the outgoing email (subject, 
+addresses, and so on) could be configured through outbound properties set in the event 
```

Comment:
We can avoid filler phrases like "and so on" with a change like this: "...properties of the outgoing email such as subject and address are configured using outbound properties set in the event..."